README.md

```diff
@@ -12,18 +35,10 @@ the CI environment), use these commands:
 
 ```
 $ crc start --cpus=6 --memory 16384
+$ export DOCKER_REPO_OVERRIDE=docker.io/username
```

Comment:
you need to export `SCALE_UP=0` for crc.

Reply:
That's not true. The default is `-1` since a while, so nothing scales unless you explicitly ask it to.